automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -185,6 +189,16 @@ def product_upgrade(product):
                     execute(satellite6_upgrade, host=sat_host)
                 else:
                     execute(satellite6_zstream_upgrade, host=sat_host)
+                # Execute following tasks post upgrade
+                # as tier1 tests are dependent on these tasks
+                certificate_url = os.environ.get('FAKE_MANIFEST_CERT_URL')
```

Comment:
To get the certificate_url via env variable raised another PR in https://github.com/SatelliteQE/robottelo-ci/pull/550